app/models/concerns/linked_data/linked_date_range.rb

```diff
@@ -3,6 +3,7 @@
 module LinkedData
   class LinkedDateRange < LinkedResource
     def local_fields
+      return {} if resource.start.blank? || resource.end.blank?
```

Comment:
Maybe this should be `&&` instead of `||` — if we had start=1900 and end=blank, wouldn't we want to display `1900-` ?

Reply:
Honestly, it should be impossible for a date range to get in there without start/end set. It's invalid (per a discussion we had before.) I'm not sure where they come from.

Reply:
OK, I'm fine with this as-is if we agreed that start and end are both required.